### DISCUSSION TITLE
Updating to  23.5.0 version

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1123] [General]
Hi
How can I update to version 23.5.0 ( by uninstalling 23.3.0 version and installing the new 23.5 one) without loosing any piece of data and tours?
Thank you

**Comment by FJBDev** [comment]
Hello,

You don't need to uninstall the 23.3.0 version (or any version). Just download and install the new 23.5.0 version and you should not lose any data and/or tours.

But as usual, it's always good to make a backup
